plugins/callback/foreman.py

```diff
@@ -264,7 +268,13 @@ def append_result(self, result):
         name = result._task.get_name()
         host = result._host.get_name()
         value = result._result
+        check_mode = result._task.check_mode
         self.items[host].append((name, value))
+        if check_mode:
+            self.check_mode = '1'
+            self._display.warning(u'Task run in check mode.')
```

Comment:
That will produce a warning for every task executed? Why?

Reply:
It is supposed to produce a warning only when the task run in check mode.
If a task runs in check mode, we would want to notify the user with a warning but if you think it is redundant, I can remove it.

Reply:
I'd argue the users knows: they asked it to be run in check mode.

Plus I'd think foreman visualizes that fact somehow in the UI, now that it knows?

Reply:
Ok, I will remove it.